scripts/projects.js

```diff
@@ -1,91 +1,89 @@
-//This is going to read project-template-outline.html and author-template.html
-//translate it for template use
-//and plug in projData.js's variables accordingly.
-//Then it will perform any necessary actions before displaying the articles, like sorting them.
-//
-//After some trial and error, I couldn't figure out how to load the external html and run it through handlebars.
-//Settling down to including the template directly into index.html for now, as to move on with the project.
-//Will revisit later.
+/**********************************************************************************\
+|  This is going to read project-template-outline.html and author-template.html    |
+|  translate it for template use and plug in projData.js's variables accordingly.  |
+|  Then it will perform any necessary actions before displaying the articles,      |
+|  like sorting them.                                                              |
+|                                                                                  |
+|  After some trial and error, I couldn't figure out how to load the external html |
+|  and run it through handlebars. Settling down to including the template directly |
+|  into index.html for now, as to move on with the project. Will revisit later.    |
+\**********************************************************************************/
 
+(function(module) {
+  function Project(itm){
+    this.title = itm.title;
+    this.category = itm.category;
+    this.description = itm.description;
+    this.imgSpoiler = itm.imgSpoiler;
+    this.pubDate = itm.pubDate
+    this.projectLink = itm.projectLink;
+    this.authorPlaceholder = '';
+    this.authors = itm.authors;
+  }
 
-//I'm reading up on the Class-06 Pair Assignment's TODOs to help
-//educate me more on the code I need to write for this Class-06 Portfolio User Stories
-//I will be using similar naming convensions so I can learn while reading and re-typing the code
-
-function Project(itm){
-  this.title = itm.title;
-  this.category = itm.category;
-  this.description = itm.description;
-  this.imgSpoiler = itm.imgSpoiler;
-  this.pubDate = itm.pubDate
-  this.projectLink = itm.projectLink;
-  this.authorPlaceholder = '';
-  this.authors = itm.authors;
-}
-
-//Instead of a global `projects = []` array, let's track this list directly on the constructor function
-//*******  I'm confused on the reasoning behind this change. *********//
-//var projects = [];
-Project.all = [];
+  //Instead of a global `projects = []` array, let's track this list directly on the constructor function
+  //*******  I'm confused on the reasoning behind this change. *********//
```

Comment:
we want to attach functions and arrays to objects, where possible. Before, this was projects = []. Now we're making it an array on the Project constructor function object. We're doing this in an effort to keep things organized and eliminate anonymous and unattached functions and arrays. make sense?